data-reporting-service/person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/patient/PatientReporting.java

```diff
@@ -0,0 +1,210 @@
+package gov.cdc.etldatapipeline.person.model.dto.patient;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.etldatapipeline.person.model.dto.DataRequiredFields;
+import gov.cdc.etldatapipeline.person.model.dto.PersonExtendedProps;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ToString(callSuper = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class PatientReporting implements PersonExtendedProps, DataRequiredFields {
+    private Long patientUid;
+    private String addlGenderInfo;
+    private String ageReported;
+    private String ageReportedUnit;
+    private String addTime;
+    private Long addUserId;
+    private String birth_sex;
+    private String dob;
+    private String currentSex;
+    private String deceasedIndicator;
+    private String deceasedDate;
+    private String generalComments;
+    private String entryMethod;
+    private String ethnicity;
+    private String unkEthnicRsn;
+    private Long lastChgUserId;
+    private String lastChgTime;
+    private String localId;
+    private String maritalStatus;
+    private String primaryOccupation;
+    private Long patientMprUid;
+    private String preferredGender;
+    private String primaryLanguage;
+    private String recordStatus;
+    private String currSexUnkRsn;
+    private String speaksEnglish;
+
+    // From Function AUTH_USR
+    private String addUserName;
+    private String lastChgUserName;
+
+    //Name from Person_Name ODSE Table
+    private String firstName;
+    private String middleName;
+    private String lastName;
+    private String nameSuffix;
+    private String aliasNickname;
+
+    //Address
+    @JsonProperty("street_address_1")
+    private String streetAddress1;
+    @JsonProperty("street_address_2")
+    private String streetAddress2;
+    private String city;
+    private String state;
+    private String stateCode;
+    private String zip;
+    private String county;
+    private String countyCode;
+    @JsonProperty("country")
+    private String countryCode;
```

Comment:
country will need the value from a different field-> home_country in the nested_address

Reply:
In mapping doc I see this for patient reporting.
`country->Address->Postal_Locator.CNTRY_CD`

Should this be from home_country?

Reply:
Addressed in mapping document